mnd/mnd/views/mnd_patient_view.py

```diff
@@ -0,0 +1,106 @@
+from django.utils.translation import ugettext as _
+
+from rdrf.views.patient_view import (
+    PatientFormMixin, AddPatientView as ParentAddPatientView, PatientEditView as ParentEditPatientView
+)
+
+
+from ..registry.patients.mnd_admin_forms import PatientInsuranceForm, PrimaryCarerForm, PreferredContactForm
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def get_section(form, section_name, section_prefix, instance):
+
+    instance = form(prefix=section_prefix, instance=instance)
+    section = (section_name, [f for f in instance.fields])
+    return instance, (section,)
+
+
+def get_form(form, request, prefix, instance=None):
+    return form(request.POST, prefix=prefix, instance=instance)
+
+
+def get_insurance_data(patient):
+    return patient.insurance_data if hasattr(patient, 'insurance_data') else None
```

Comment:
return getattr(patient, 'insurance_data', None) ?

Applies to the other similar getters below.

Reply:
Updated.